src/psij/resource_spec.py

```diff
@@ -1,9 +1,17 @@
 from abc import ABC, abstractmethod
-from typing import Optional, Tuple
+from typing import Optional, List
 
 from psij.exceptions import InvalidJobException
 
 
+def _nulls(objs: List[object]) -> int:
+    s = 0
+    for e in objs:
+        if e is None:
+            s += 1
+    return s
+
```

Comment:
Isn't this the same as `list.count(value)`?
```py
>>> l = [1, None, 2, None, 3]
>>> l.count(None)
2
```

Reply:
Silly me. Yes. Will update.